### DISCUSSION TITLE
Wired internet connection does not work !

### DISCUSSION THREAD

**Discussion by claudeLessard** [Joshua-Riek/ubuntu-rockchip#62] [General]
I cannot connect to a wired internet connection ! Any solutions ?

**Comment by Joshua-Riek** [comment]
That is weird, wired internet connections have been working perfectly for me. Can you verify that your ethernet cable works with another device, such as a laptop? Also, what version are you using?

**Reply by claudeLessard** [reply]
With another operating system and the same internet cable connect to the same swith ethernet it works. With your operating system when I click on the network connection I get the error message: the connection could not be established with this configuration (I tried dhcp and manual same problem). I am a programmer not a beginner

**Reply by Joshua-Riek** [reply]
Weird. I'm going to download the latest release again and see if i get this on my side.

**Reply by Joshua-Riek** [reply]
I don't have any issue with a wired ethernet connection on my side. I used a fresh install of v1.8.
![Screenshot from 2023-03-09 12-17-51](https://user-images.githubusercontent.com/10427125/224105408-6c2fdfcc-8a88-439d-b926-f8df92d60170.png)

**Reply by claudeLessard** [reply]
I flashed another sdcard and it works ! thanks !

**Reply by Joshua-Riek** [reply]
Glad to hear!

**Reply by claudeLessard** [reply]
Hello if I install kde will I lose the optimizations you have brought to your operation system

**Reply by Joshua-Riek** [reply]
I made optimizations specifically for gnome and wayland, I have no idea how kde would perform. You can give it a try but be warned, it could very well break things.

**Reply by claudeLessard** [reply]
ok I will not try it. Are you going to make a version with kde in the futur ?

**Reply by Joshua-Riek** [reply]
I can't guarantee that I will. But I might in the future, depending on how much free time I have and the state of kde. If it's a buggy mess with this board I might not bother.

**Reply by claudeLessard** [reply]
Thanks for the effort you made for this operating system.

**Reply by claudeLessard** [reply]
Hi, I installed kde but it won't start. I did not loose anything in ubuntu (your uptimizations).
